trustmanager/keys.go

```diff
@@ -210,7 +210,7 @@ func checkValidity(block *pem.Block) (string, error) {
 		// check if the key is missing a gun header or has an empty gun and error out since we don't know what gun it belongs to
 		if block.Headers["gun"] == "" {
 			logrus.Warnf("failed to import key (%s) to store: Cannot have canonical role key without a gun, don't know what gun it belongs to", block.Headers["path"])
-			return "", errors.New("invalid key pem block")
+			return "", errors.New("invalid key pem block.")
```

Comment:
We're not perfect in execution but generally errors shouldn't end in punctuation c.f. https://github.com/golang/go/wiki/CodeReviewComments#error-strings